Fix APPLICATION_ROOT example in docs (#1270)
spiral-project/ihatemoney: docs/configuration.md

## APPLICATION_ROOT

> If empty, ihatemoney will be served at domain root (e.g:

Author: Perhaps this should be changed to "by default" since the default value is "/"

```suggestion
By default, ihatemoney will be served at domain root (e.g:
```
